frontend/apps/reader/modules/readerfont.lua

```diff
@@ -82,8 +82,28 @@ function ReaderFont:init()
         })
         face_list[k] = {text = v}
     end
+    self.face_table[#self.face_table].separator = true
+    table.insert(self.face_table, {
+        text = _("Complementary standard fallback fonts"),
+        checked_func = function()
+            return G_reader_settings:nilOrTrue("complementary_fallback_fonts")
+        end,
+        callback = function()
+        G_reader_settings:flipNilOrTrue("complementary_fallback_fonts")
+            self.ui.document:setupFallbackFontFaces()
+            self.ui:handleEvent(Event:new("UpdatePos"))
+        end,
+        help_text = T(_([[
+Enable additional fallback fonts, for the most complete scripts and languages coverage.
+Complementary fonts, used in this order:
```

Comment:
Additional?

(Complementary is... clunky in English, possibly because it's so close to complimentary, which is another thing entirely ^^)

Reply:
As we have "additional" in the first sentence, may be for the 2nd sentence just:
`These fonts will be used in this order:`

Reply:
I guess your warning applies to the menu too:
`Complementary standard fallback fonts`
Suggestion for less heavy wording for this one? :)

Reply:
`Recommended` maybe? Not such a fan of that either, but it mostly works ;p.